qemu.go

```diff
@@ -106,6 +108,11 @@ const (
 	maxDevIDSize = 31
 )
 
+const (
+	// NVDIMM device needs memory space (1024MB is suggested by @grahamwhaley)
```

Comment:
+1 for the comment but please can you add a link to an issue/PR rather than mentioning individuals.